internal/cli/roles_permissions.go

```diff
@@ -0,0 +1,254 @@
+package cli
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	roleAPIIdentifier = Flag{
+		Name:       "API",
+		LongForm:   "api-id",
+		ShortForm:  "a",
+		Help:       "API Identifier.",
+		IsRequired: true,
+	}
+
+	roleAPIPermissions = Flag{
+		Name:       "Permissions",
+		LongForm:   "permissions",
+		ShortForm:  "p",
+		Help:       "Permissions.",
+		IsRequired: true,
+	}
+)
+
+func rolePermissionsCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "permissions",
+		Short:   "Manage permissions within the role resource",
+		Long:    "Manage permissions within the role resource.",
+		Aliases: []string{"permission"},
```

Comment:
Why that alias in particular? We don't have any other aliases (nor commands)  in singular.